github-status/handler.go

```diff
@@ -139,15 +139,12 @@ func buildPublicStatusURL(status string, statusContext string, event *sdk.Event)
 				url = publicURL + "function/" + serviceValue
 			}
 		} else { // For context Stack on success the gateway url is used
-			if len(gatewayPrettyURL) > 0 {
-				// https://user.get-faas.com/function
-				url = strings.Replace(gatewayPrettyURL, "user", event.Owner, 1)
-				url = strings.Replace(url, "function", "", 1)
-			} else if len(publicURL) > 0 {
+			if len(publicURL) > 0 {
 				if strings.HasSuffix(publicURL, "/") == false {
 					publicURL = publicURL + "/"
 				}
-				url = publicURL
+				publicURL = publicURL + "dashboard/" + event.Owner + "/"
+				url = strings.Replace(publicURL, "://", "://system.", 1)
```

Comment:
I think `cloud.o6s.io` is configurable and we don't know how it will be configured in order to replace the first prefix. AFAIK, `system` prefix is for the whole public url.